### PR TITLE
SCT-47 Support Maven Wrapper

### PR DESCRIPTION
**Pull request by cppwfs**
resolves spring-cloud/spring-cloud-task#47

### CODE REVIEW

File: .mvn/maven.config [hunk 1]
```diff
@@ -0,0 +1 @@
+-DaltSnapshotDeploymentRepository=repo.spring.io::default::https://repo.spring.io/libs-snapshot-local -s settings.xml -P spring
```
Comment (markpollack) [inline .mvn/maven.config]: Should be .settings.xml

Comment (cppwfs) [inline .mvn/maven.config]: The spring task project the settings.xml file does not have the `.` prefix.

Comment (trisberg) [inline .mvn/maven.config]: it should use .settings.xml to be consistent with other  projects

**Comment by trisberg** [comment]
Can we also add a default profile that includes the repos like here -- https://github.com/spring-cloud/spring-cloud-stream-modules/blob/master/pom.xml#L212

File: .mvn/maven.config (hunk 1 above)
Comment (mminella) [inline .mvn/maven.config]: -1 for changing the name to `.settings.xml`.  If we are going to require it's use, it shouldn't be hidden.  Either set it up so it's not needed, or don't hide it IMHO.

Comment (cppwfs) [inline .mvn/maven.config]: Will do

**Comment by mminella** [comment]
Squashed and merged as 30497e62584879130c80b2fe9431c864f265c35a.

**Pull request closed without merging**